sway fails to compile

This is after running make a second time, since I get too much scrollback otherwise

I seem to get this output .


This is on 32 bit. Let me know if it's something on my end
```
ninja: Entering directory `build'
[1/8] Linking target sway/sway.
[2/8] Compiling C object 'swaybar/swaybar@exe/status_line.c.o'.
FAILED: swaybar/swaybar@exe/status_line.c.o 
cc  -Iswaybar/swaybar@exe -Iswaybar -I../swaybar -Iinclude -I../include -I/opt/include/cairo -I/opt/include/pixman-1 -I/opt/include/freetype2 -I/opt/include/harfbuzz -I/opt/include/glib-2.0 -I/opt/lib/i386-linux-gnu/glib-2.0/include -I/opt/include/libdrm -I/opt/include -I/usr/include/uuid -I/usr/include/libpng16 -I/usr/include -I/opt/include/gdk-pixbuf-2.0 -I/opt/include/json-c -I/opt/include/pango-1.0 -I/usr/include/fribidi -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -std=c11 -Wno-unused-parameter -Wno-unused-function -Wno-unused-result '-DSYSCONFDIR="/etc"' '-DSWAY_VERSION="1.0-alpha.1-67-g54ae3947 (" __DATE__ ", branch '"'"'master'"'"')"' -Os -pthread -MMD -MQ 'swaybar/swaybar@exe/status_line.c.o' -MF 'swaybar/swaybar@exe/status_line.c.o.d' -o 'swaybar/swaybar@exe/status_line.c.o' -c ../swaybar/status_line.c
In file included from /usr/include/fcntl.h:25:0,
                 from ../swaybar/status_line.c:2:
/usr/include/features.h:244:53: error: operator '&&' has no right operand
      || (defined _POSIX_C_SOURCE && _POSIX_C_SOURCE >= 1) \
                                                     ^~
/usr/include/features.h:249:48: error: operator '&&' has no right operand
 #if defined _POSIX_C_SOURCE && _POSIX_C_SOURCE >= 2 || defined _XOPEN_SOURCE
                                                ^~
../swaybar/status_line.c: In function ‘status_line_init’:
../swaybar/status_line.c:120:17: warning: implicit declaration of function ‘fdopen’ [-Wimplicit-function-declaration]
  status->read = fdopen(status->read_fd, "r");
                 ^~~~~~
../swaybar/status_line.c:120:15: warning: assignment makes pointer from integer without a cast [-Wint-conversion]
  status->read = fdopen(status->read_fd, "r");
               ^
../swaybar/status_line.c:121:16: warning: assignment makes pointer from integer without a cast [-Wint-conversion]
  status->write = fdopen(status->write_fd, "w");
                ^
../swaybar/status_line.c: In function ‘status_line_free’:
../swaybar/status_line.c:128:2: warning: implicit declaration of function ‘kill’ [-Wimplicit-function-declaration]
  kill(status->pid, SIGTERM);
  ^~~~
[3/8] Compiling C object 'swaylock/swaylock@exe/password.c.o'.
[4/8] Compiling C object 'swaylock/swaylock@exe/seat.c.o'.
[5/8] Compiling C object 'swaylock/swaylock@exe/render.c.o'.
ninja: build stopped: subcommand failed.
PackageInstallSource finished in 0 seconds with result 1

```

--- a/swaybar/status_line.c
+++ b/swaybar/status_line.c
@@ -1,4 +1,4 @@
-#define _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 199309L
 #include <fcntl.h>
 #include <json-c/json.h>
 #include <stdlib.h>

